production/ksonnet/loki/ruler.libsonnet

```diff
@@ -9,7 +9,7 @@ local k = import 'ksonnet-util/kausal.libsonnet';
 
   ruler_args:: $._config.commonArgs {
     target: 'ruler',
-  } + if $._config.using_boltdb_shipper then {
+  } + if $._config.using_boltdb_shipper && !$._config.using_tsdb_shipper then {
     // Use PVC for caching
     'boltdb.shipper.cache-location': '/data/boltdb-cache',
```

Comment:
IMO these lines can be removed altogether, since the cache location is already defined in the config, see your additions (lines 10-19) in `production/ksonnet/loki/shipper.libsonnet`. Wdyt?

Reply:
It looks like here the config is being passed to the ruler component, I'm not super familiar with this component so to be honest I'm not sure if this is actually being used by the ruler, if you think that it isn't then happy to delete it?

Reply:
The global config is picked up by all components. So it's safe to delete it here.

Reply:
I see makes sense, I have removed this config altogether here then.